app/src/main/cpp/skyline/services/nvdrv/devices/nvhost_ctrl.h

```diff
@@ -31,61 +31,40 @@ namespace skyline {
                 Cancelled = 5,
             };
 
-            NvHostEvent(const DeviceState &state);
+            SyncpointEvent(const DeviceState &state);
 
+            std::recursive_mutex mtx;
```

Comment:
We generally use `mutex` rather than `mtx`, I'll make this change global but you should add it here. A comment regarding what the mutex helps protect would be nice as well.